client/components/deposits-overview/utils.ts

```diff
@@ -38,6 +40,7 @@ export const getNextDeposit = (
 		id: nextScheduled.id,
 		date: nextScheduled.date ?? 0,
 		status: nextScheduled.status ?? 'estimated',
-		amount: formatCurrency( nextScheduled.amount ?? 0, currency ),
+		formattedAmount: formatCurrency( nextScheduled.amount ?? 0, currency ),
+		amount: nextScheduled.amount ?? 0,
```

Comment:
I had to introduce an raw `amount` figure so I could use the figure in logic without having to reverse format it. 

If it turns out we're not suppose to use the next deposit amount this can be reverted.

Reply:
I _think_ we can revert this but need to get the logic right first!